src/components/HomeWidgets/Widget1.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import Widget1 from "./Widget1";
+
+export default {
+  title: 'Home/Widget1',
+  component: Widget1,
+}
+
+export const Widget1Component = () => (
```

Comment:
Would recommend better names than just Widget1Component. But I'm sure this is a placeholder.

Reply:
yeah I'll change that when I build this component out